README.md

```diff
@@ -34,15 +34,41 @@ pipx ensurepath
 
 Upgrade pipx with `brew update && brew upgrade pipx`.
 
-Otherwise, install via pip (requires pip 19.0 or later):
+### On Linux, install via pip (requires pip 19.0 or later)
 
 ```
 python3 -m pip install --user pipx
 python3 -m pipx ensurepath
 ```
+Upgrade pipx with `python3 -m pip install --user -U pipx`.
```

Comment:
Perhaps we should replace --user with https://pypi.org/project/pipx-in-pipx/ 🤔

Reply:
My mind is now in an infinite loop

Reply:
Instead of adding documentation to this I thought it would even be better to have the installation ask for and actually adding the path variable instead of issuing a warning. Don't know if that's doable.

Reply:
Or https://github.com/uranusjr/pipx-standalone 🙂

Reply:
> 
> 
> Or we could incorporate https://github.com/uranusjr/pipx-standalone into the main repo and the release process 🙂

Scoop looks nice! But I would consider this an extra. Not as the default solution tbh.

Reply:
Yeah, it's also a Windows-only thing :thinking: while pipx-in-pipx is cross-platform compatible solution.

Reply:
FWIW we have Homebrew five lines into the documentation, and it is very much not universal.

Reply:
But can't we -in the meantime- just improve on the installation instruction of what we have now ? 
I haven't looked into pipx-in-pipx but it is -like scoop I guess- yet another dependency.

Reply:
It's actually not. It's just a wrapper to install pipx with pipx :blush: It just requires pip and python which all platforms should already have.

Reply:
Ok. So what do you suggest? Do you prefer this to become the default way of installing pipx? 
In any case, this is out of my knowledge scope to have a proper opinion about.

My only fear is -right or wrong- we're solving one thing and potentially breaking a lot of other. Assuming we only have issues installing on windows..

Reply:
I personally would prefer it to become the standard :thinking:

Reply:
Knowing your merits inside the python community I have full trust adopting your personal preference and will happily set aside my fears. 

But this is definitely not my choice to make. So close this PR then ?
I will start playing around with pipx-in-pipx.

Reply:
Or should pipx fully absorb pipx-in-pipx ?

Reply:
I don't think pipx can absorh pipx-in-pipx, at best we'd look at pipx maintainers adopting it and moving it under PyPA. I don't think you need to abandon this PR, the changes are still great, all I'm saying is that perhaps we could switch from using ``--user`` to use ``pipx-in-pipx`` 🤔